test/MeshKernelNETTest/Api/MeshKernelCurvilinearTest.cs

```diff
@@ -771,5 +772,42 @@ public void CurvilinearInsertFaceThroughAPI()
                 }
             }
         }
+
+        [Test]
+        public void CurvilinearGetSmoothnessThroughAPI()
+        {
+            // Setup
+            using (var api = new MeshKernelApi())
+            using (DisposableCurvilinearGrid grid = CreateCurvilinearGrid(5, 5, 10, 10))
+            {
+                var id = 0;
+                var curvilinearGrid = new DisposableCurvilinearGrid();
```

Comment:
the instance created here is not disposed of: the CurvilinearGridGetData will return a new instance. Prefer to initialize to `null` here and in the finally clause call `curvilinearGrid?.Dispose()`